pkg/controller/scyllacluster/resource.go

```diff
@@ -110,6 +110,36 @@ func MemberService(sc *scyllav1.ScyllaCluster, rackName, name string, oldService
 		}
 	}
 
+	var internalTrafficPolicy *corev1.ServiceInternalTrafficPolicy
```

Comment:
would it be more readable to start by defining a Service and use the fields directly then extra vars?

Reply:
yep, looks better. thanks